app/src/main/java/com/example/habitsmasher/ui/dashboard/EditHabitFragment.java

```diff
@@ -26,7 +26,6 @@
 
 
 public class EditHabitFragment extends DialogFragment implements DatePickerDialog.OnDateSetListener{
-    private static final String HEADER = "Edit Dialog";
```

Comment:
This was deleted and moved because I decided to pull from the resource file rather than a hard coded string.